packages/rspack/src/builtin-plugin/LightningCssMiminizerRspackPlugin.ts

```diff
@@ -1,26 +1,72 @@
 import {
 	BuiltinPluginName,
+	type RawLightningCssBrowsers,
 	type RawLightningCssMinimizerRspackPluginOptions
 } from "@rspack/binding";
 
+import browserslist from "browserslist";
+import {
+	type Drafts,
+	type FeatureOptions,
+	type NonStandard,
+	type PseudoClasses,
+	type Targets,
+	browserslistToTargets,
+	toFeatures
+} from "../builtin-loader/lightningcss";
+import type { AssetConditions } from "../util/assetCondition";
 import { create } from "./base";
 
-export type LightningCssMinimizerRspackPluginOptions =
-	Partial<RawLightningCssMinimizerRspackPluginOptions>;
+export type LightningCssMinimizerRspackPluginOptions = {
+	test?: AssetConditions;
+	include?: AssetConditions;
+	exclude?: AssetConditions;
+	removeUnusedLocalIdents?: boolean;
+	minimizerOptions?: {
+		errorRecovery?: boolean;
+		targets?: Targets | string[] | string;
+		include?: FeatureOptions;
+		exclude?: FeatureOptions;
+		draft?: Drafts;
+		nonStandard?: NonStandard;
+		pseudoClasses?: PseudoClasses;
+		unusedSymbols?: string[];
+	};
+};
 
 export const LightningCssMinimizerRspackPlugin = create(
 	BuiltinPluginName.LightningCssMinimizerRspackPlugin,
 	(
 		options?: LightningCssMinimizerRspackPluginOptions
 	): RawLightningCssMinimizerRspackPluginOptions => {
+		const { include, exclude, draft, nonStandard, pseudoClasses } =
+			options?.minimizerOptions ?? {};
+		const targets = options?.minimizerOptions?.targets ?? "Chrome 60"; // last not support es module chrome version
```

Comment:
Why use `Chrome 60` rather than es6 here

Reply:
Can't use `es6` in browserslist

Reply:
We can use `fully supports es6` https://github.com/browserslist/browserslist

![image](https://github.com/user-attachments/assets/0668b2df-4eea-4d10-9ce4-6a5f54c5e9db)

Reply:
https://browsersl.ist/#q=fully+supports+es6